### DISCUSSION TITLE
<MediaController> takes focus?

### DISCUSSION THREAD

**Discussion by intelligence** [muxinc/media-chrome#1034] [Q&A]
After some thorough head scratching I've tracked down that introducing <MediaController> in my app makes it takes focus in a carousel slider. I've got a Mux video on slide 2 and whenever <MediaController> is there it will jump to that slide. Disabling the carousel feature and simply having a horizontal scroll produces the same result. Removing <MediaController> or setting `display: none` on it, fixes it.

Any idea what might cause this behaviour?

**Comment by luwes** [comment]
could you provide an online demo or code sandbox? that will help a lot with debugging

have you tried changing the `tabindex` attribute?

**Comment by intelligence** [comment]
Hey! I'm in the middle of getting this project production ready so I'll get back to you later.

I tried settings `tabindex` to `-1` and `500` but that didn't change the behaviour.
